### DISCUSSION TITLE
Question Regarding challenge 1 i.e Machine learning training assistant

### DISCUSSION THREAD

**Discussion by pranavvp16** [camicroscope/GSOC#22] [Q&A]
The challenge mentions to create a tensorflow-js frontend application that gives some form analysis on user supplied image, but what kind of image should the image be pathology related ?

**Comment by tcpan** [comment]
@pranavvp16, sorry for the delay in responding.  Ideally, yes, pathology or microscopy images would be best, but something with lots of small objects and textures would illustrate the point.
